tike/tike.py

```diff
@@ -83,7 +83,7 @@ def _combined_interface(obj, obj_min,
     return None
 
 
-def admm(obj=None, voxelsize=None,
+def admm(obj=None, voxelsize=1.0,
          data=None,
          probe=None, theta=None, h=None, v=None, energy=None,
          niter=1, rho=0.5, gamma=0.25,
```

Comment:
Are the args explained in func docs?

Reply:
Good catch. I am adding a docstring for `voxelsize` and `niter`. `rho`, `gamma` are actually parameters for `ptycho.grad`. Those need to be moved into `pkwargs` when we do the thing where we reduce the number of function parameters by packing everything into dictionaries.

Reply:
```rho``` is actually an ADMM parameter. On the ptychography side its default should to zero.

Reply:
`rho` cannot be zero for `pytcho.grad`. That would cause a divide by zero error on [this line](https://github.com/carterbox/tike/blob/92c8a639670dc0bdd6e30eb5175a26df9cc57a52/tike/ptycho.py#L388-L391). It could be set to 1 by default?

Reply:
I'd default it with a very small number instead of one. 1e-12?

Reply:
We can rearrange the math to prevent division by zero:
```python
+ gamma * rho * (reg - lamda / rho) # this is the original
+ gamma * (reg * rho - lamda) # this is rho multiplied through
```
@slnsrydn, said it was allowed.

Reply:
Yes. This is the way to do it.